lib/openstudio-standards/standards/necb/common/btap_data.rb

```diff
@@ -499,7 +499,8 @@ def space_table(model, cost_result)
   def climate_data
     # Store Geography Data
     geography_data = {}
-    geography_data['location_necb_hdd'] = @standard.get_necb_hdd18(@model)
+    geography_data['location_necb_hdd'] = @standard.get_necb_hdd18(model: @model, necb_hdd: true)
```

Comment:
why both fields?

Reply:
I should fix that.  I evidently didn't look closely enough at what was already there when I wrote that.

Reply:
Thanks for pointing that out.  I removed the redundant hdd report.